server/routes.go

```diff
@@ -51,5 +51,11 @@ func getRoutes(app *middleware.App) []Route {
 			"/healthcheck/ping/",
 			controllers.Ping,
 		},
+		{
+			"Project Registration",
+			"POST",
+			"/project/add/",
+			middleware.WithLogin(middleware.WrapApp(app, controllers.RegisterProject)),
```

Comment:
does the route need to have the /add/ ? Ig `POST /project/` itself refers to adding a project

Reply:
Backend v1 has `/add`

Reply:
yes

Reply:
as long as the method are different, they dont overlap